pyproject.toml

```diff
@@ -76,13 +76,15 @@ doc = [
     "ansys-dpf-core==0.10.1",
     "ansys-mapdl-reader==0.54.1",
     "ansys-tools-visualization-interface==0.4.4",
-    "ansys-sphinx-theme==1.0.11",
+    "ansys-sphinx-theme@git+https://github.com/ansys/ansys-sphinx-theme.git@fix/test-cheatsheet",
```

Comment:
I dont like this.... Doesn't it work with latest release?

Reply:
This will be removed prior to merge the PR

Reply:
A release should occur shortly